client/src/app/core/pdf-services/pdf-worker.worker.ts

```diff
@@ -35,7 +35,7 @@ function applyLayout(content: any): void {
         if (Array.isArray(section)) {
             applyLayout(section);
         } else {
-            if (!!section.layout) {
+            if (!!section && !!section.layout) {
```

Comment:
This threw an error for me as soon as I exported a comment field which was empty on some motions